design-system/src/_includes/pattern-library/foundations/foundations-buttons/buttons.html

```diff
@@ -1,19 +1,74 @@
 
-<div class="coop-u-padding-resp-16-8 coop-u-margin-b-16">
-    <button class="coop-btn coop-btn--primary">Primary button</button>
-</div>
-<div class="coop-u-padding-resp-16-8 coop-u-margin-b-16">
-    <button class="coop-btn">Button</button>
-</div>
-<div class="coop-u-padding-resp-16-8 coop-u-margin-b-16">
+<form class="coop-form">
+  <h3>Primary button</h3>
+
+  <div class="coop-form__row">
+    <button class="coop-btn coop-btn--primary">Join Co-op for £1</button>
```

Comment:
How come the modifier to make it primary?

Reply:
Don't know, this was @matt-tyas's original class 😊

I think it's thought of as more of a "Call to action" button as the notes say:

>The primary button should only be used once per page.

Reply:
Fair enough 👍 

Guess in the future, all buttons should be modified to make intent clearer.